lib/solidus_segment/analytics.rb

```diff
@@ -4,8 +4,27 @@ module SolidusSegment
   class Analytics
     attr_reader :backend
 
-    def initialize(backend: SolidusSegment.configuration.segment_backend)
+    def initialize(user: nil,
+                   anonymous_id: nil,
+                   backend: SolidusSegment.configuration.segment_backend)
```

Comment:
I think that `user` and `anonymous_id` would be better passed to `identify` method to be able to use the clients for multiple calls if needed and to keep consistency with Segment's API.

Reply:
The goal of this class is to identify and track the user, I've thought the interface should be like this:
```ruby
analytics = SolidusSegment::Analytics.new(user: user)
analytics.track_add_to_cart(product: product)
analytics.track_order_complete(order: order)
```

Reply:
👌